src/operators.jl

```diff
@@ -183,8 +185,55 @@ end
 function *{T<:Real}(g::DiGraph, v::Vector{T})
```

Comment:
I am copying the tests into a LightGraphs with @sbromberger 's approach.
Does `if_installed(m) do ... end` work if you try to use functions and macros defined in `m` in the function `f`? I think macros break it because they are evaluated before the the try catch is executed.